dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/property/DssRestChannelProperty.java

```diff
@@ -17,10 +17,11 @@
     private static final long serialVersionUID = -6904414304979344025L;
     private final String host;
     private final int port;
-
+    private final DssLogLevel dssLogLevel;
     @Builder
-    private DssRestChannelProperty(String host, int port) {
+    private DssRestChannelProperty(String host, int port, DssLogLevel dssLogLevel) {
         this.host = host;
         this.port = port;
+        this.dssLogLevel = dssLogLevel;
```

Comment:
@vincentj2 
Hi~

It looks you do not set default value of dssLogLevel and cause ci failed

Reply:
hi

yes.. that's my mistake.   I'll fit it!

Reply:
I fix line 25 
` this.dssLogLevel = dssLogLevel; `
to
` this.dssLogLevel = DssLogLevel.DEBUG; `

Is it okey?

Reply:
Nope.
Following better, i think.
``` java
this.dssLogLevel = Objects.nonNull(dssLogLevel) ? dssLogLevel : DssLogLevel.DEBUG;
```

Reply:
I think that will be better!!
thank you :)